pipelines/library/src/common/VersionInfo.groovy

```diff
@@ -73,23 +100,34 @@ class VersionInfo {
     }
 
     private boolean matchPre223(versionString) {
+        context.println "[INFO] Attempting to match pre223 regex..."
+
         final pre223regex = "jdk\\-?(?<version>(?<major>[0-8]+)(u(?<update>[0-9]+))?(-b(?<build>[0-9]+))(_(?<opt>[-a-zA-Z0-9\\.]+))?)"
         final matched = versionString =~ /${pre223regex}/
 
         if (matched.matches()) {
+            context.println "[SUCCESS] pre223 regex matched!"
             major = or0(matched, 'major')
             minor = 0
             security = or0(matched, 'update')
             build = or0(matched, 'build')
-            if (matched.group('opt') != null) opt = matched.group('opt')
+
+            if (matched.group('opt') != null) {
+                opt = matched.group('opt')
+                context.println "[SUCCESS] pre223 regex group (opt) matched to ${opt}"
+            }
+
             version = matched.group('version')
             return true
         } else {
+            context.println "[WARNING] Failed to match pre223 regex! Attempting to matchAltPre223 regex..."
             return matchAltPre223(versionString)
         }
     }
 
     private boolean match223(versionString) {
+        context.println "[INFO] Attempting to match223 regex..."
```

Comment:
Probably a little duplication of logging going on in this PR, maybe reduce some duplicate looking lines.

Reply:
I'm not seeing any logging duplication in the output?
```
13:03:29  [INFO] ATTEMPTING TO PARSE PUBLISH_NAME: 1.8.0_272-202010271236-b10
[Pipeline] echo
13:03:29  [INFO] Attempting to match pre223 regex...
[Pipeline] echo
13:03:29  [WARNING] Failed to match pre223 regex! Attempting to matchAltPre223 regex...
[Pipeline] echo
13:03:29  [SUCCESS] matchAltPre223 regex matched!
[Pipeline] echo
13:03:29  [INFO] Parsing additional group of matchAltPre223 regex...
[Pipeline] echo
13:03:29  [SUCCESS] matchAltPre223 regex group (opt) matched to 202010271236
[Pipeline] echo
13:03:29  [SUCCESS] matchAltPre223 regex group (build) matched to 10
[Pipeline] echo
13:03:29  [INFO] FINISHED PARSING PUBLISH_NAME:
[Pipeline] echo
13:03:29  major = 8
[Pipeline] echo
13:03:29  minor = 0
[Pipeline] echo
13:03:29  security = 272
[Pipeline] echo
13:03:29  build = 10
[Pipeline] echo
13:03:29  opt = 202010271236
[Pipeline] echo
13:03:29  version = 1.8.0_272-202010271236-b10
[Pipeline] echo
13:03:29  pre = null
[Pipeline] echo
13:03:29  adopt build number = 0
[Pipeline] echo
13:03:29  semver = 8.0.272+10.0.202010271236
```
Any less than this would result in an inaccurate picture of the VersionInfo and the method that was used to retrieve it